container-scan/action.yaml

```diff
@@ -1,152 +1,67 @@
         build-args: ${{ inputs.build-args }}
-        secrets: ${{ inputs.secrets }}
 
-    - name: "Determining image name"
-      shell: bash
-      id: set_image_name
+    - name: Determine Image Name
       run: |
-        if [ "${{ inputs.enable-docker-build }}" == "true" ]; then
-          echo "::set-output name=image_name::${{ steps.docker-build.outputs.image-name }}"
+        if [ "${{ inputs.enable-docker-build }}" = "true" ]; then
+          echo "IMAGE_NAME=${{ steps.docker-build.outputs.image-name }}" >> $GITHUB_ENV
         else
-          echo "::set-output name=image_name::${{ inputs.image-name }}"
+          echo "IMAGE_NAME=${{ inputs.image-name }}" >> $GITHUB_ENV
         fi
+      shell: bash
 
-    - name: Scan container image for vulnerabilities using Lacework
+    - name: Scan Container Image
+      id: scan
       uses: lacework/lw-scanner-action@v1.4.3
       with:
         LW_ACCOUNT_NAME: ${{ inputs.lw-account-name }}
         LW_ACCESS_TOKEN: ${{ inputs.lw-access-token }}
-        IMAGE_NAME: ${{ steps.set_image_name.outputs.image_name }}
+        IMAGE_NAME: ${{ env.IMAGE_NAME }}
         IMAGE_TAG: ${{ inputs.image-tag }}
         SAVE_RESULTS_IN_LACEWORK: true
-        RESULTS_IN_GITHUB_SUMMARY: true
-        PRETTY_OUTPUT: true
-
-    - name: Check if Lacework scan results file exist
-      id: check-results
-      run: |
-        if [ -f results.stdout ]; then
-          echo "Scan results file (results.stdout) exists"
-          exit 0
-        else
-          echo "Scan results file (results.stdout) does not exist"
-          exit 1
-        fi
-      shell: bash
-
-    - name: Change formatting for PR
-      if: always()
-      run: |
-        echo "## Lacework Inline Scanner Results" > pr-results.md
-        echo "<details><summary>Click to expand</summary>" >> pr-results.md
-        echo "<pre>" >> pr-results.md
-        cat results.stdout >> pr-results.md
-        echo "</pre>" >> pr-results.md
-        echo "</details>" >> pr-results.md
-      shell: bash
+        ADDITIONAL_PARAMETERS: "-j"
 
-    - name: Check for previous report comment
-      id: fc
-      if: github.event_name == 'pull_request' && github.event.pull_request.number != ''
-      uses: peter-evans/find-comment@v3
-      with:
-        issue-number: ${{ github.event.pull_request.number }}
-        comment-author: "github-actions[bot]"
-        body-includes: "Lacework Inline Scanner Results"
-
-    - name: Delete previous test report comment
-      if: github.event_name == 'pull_request' && steps.fc.outputs.comment-id != ''
-      uses: winterjung/comment@v1
-      with:
-        type: delete
-        comment_id: ${{ steps.fc.outputs.comment-id }}
-        token: ${{ inputs.github-token }}
-
-    - name: Comment PR
-      if: github.event_name == 'pull_request'
-      uses: thollander/actions-comment-pull-request@v3
-      with:
-        filePath: pr-results.md
-        mode: recreate
-        comment_tag: to_recreate
```

Comment:
Any reason why the publishing of container scan report is removed? 🤔

Reply:
@gopkri24 moving this to the reusable workflow. reasoning behind it was to have a cleaner separation of concerns and a centralized token management.